packages/env-tests/src/scaffold.ts

```diff
@@ -21,22 +29,28 @@ export async function fundAccount(
   })
   context.kit.connection.addAccount(root.privateKey)
 
-  const stableToken = await context.kit.contracts.getStableToken()
-
-  const rootBalance = await stableToken.balanceOf(root.address)
-  if (rootBalance.lte(value)) {
-    logger.error({ rootBalance: rootBalance.toString() }, 'error funding test account')
-    throw new Error(
-      `Root account ${root.address}'s balance (${rootBalance.toPrecision(
-        4
-      )}) is not enough for transferring ${value.toPrecision(4)}`
+  for (const stableToken of context.stableTokensToTest) {
+    let stableTokenAddress = await context.kit.registry.addressFor(
+      StableTokenToRegistryName[stableToken] as CeloContract
     )
-  }
-  const receipt = await stableToken
-    .transfer(recipient.address, value.toString())
-    .sendAndWaitForReceipt({ from: root.address, feeCurrency: stableToken.address })
+    let stableTokenContract = newStableToken(context.kit.web3, stableTokenAddress)
+    let stableTokenInstance = new StableTokenWrapper(context.kit, stableTokenContract)
```

Comment:
This feels like a pattern. Maybe we can move it to a helper: `getStableTokenWrapper(kit, 'cUSD')` and use it in other places as well.

Reply:
Makes sense, I created a helper function